src/Api/Service/MyParcelApiService.php

```diff
@@ -6,18 +6,19 @@
 
 use Composer\InstalledVersions;
 use GuzzleHttp\Client;
+use MyParcelNL\Pdk\Base\Pdk;
 
 /**
  * This will replace the SDK one day...
  */
 class MyParcelApiService extends AbstractApiService
 {
-    public const  PACKAGE_NAME     = 'myparcelnl/pdk';
     private const DEFAULT_BASE_URL = 'https://api.myparcel.nl';
     private const DEFAULT_CONFIG   = [
-        'baseUrl'   => self::DEFAULT_BASE_URL,
-        'client'    => Client::class,
-        'userAgent' => null,
+        'apiKey'     => '',
```

Comment:
nooit lege strings gebruiken, gewoon null
```suggestion
        'apiKey'     => null,
```